R/geom-point.r

```diff
@@ -98,6 +98,14 @@ geom_point <- function(mapping = NULL, data = NULL,
                        na.rm = FALSE,
                        show.legend = NA,
                        inherit.aes = TRUE) {
+  dots <- list(...)
```

Comment:
I don't think this is the right place to do translation - it would be better to do so during drawing.